### PR TITLE
[TTAHUB-693] Update Creator Role for Display & CSV

### PR DESCRIPTION
**Pull request by AdamAdHocTeam**
## Description of change

The alerts and approved AR tables and their CSV exports now show the creator and role set on the report.

NOTE: 
- If the report is in the alerts table and a creator role has NOT been set we only show the creator name.
- If the report is in the approved AR table and is missing a creator role we will show the creator and all roles (historical).

## How to test

View the creator column in both alerts and approved AR tables and csv exports.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-693

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [x] Update JIRA ticket status

### CODE REVIEW

File: src/models/activityReport.js [hunk 1]
```diff
@@ -275,6 +275,20 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: true,
       type: DataTypes.ENUM(Object.keys(USER_ROLES).map((k) => USER_ROLES[k])),
     },
+    creatorName: {
+      type: DataTypes.VIRTUAL,
+      get() {
+        // If a creator role is not set but the report is in ALERTS.
+        // We want to show the name with no roles.
+        if (this.creatorRole || this.calculatedStatus !== REPORT_STATUSES.APPROVED) {
```
Comment (AdamAdHocTeam) [inline src/models/activityReport.js:283]: I'm not 100% sold on the second half of this statement. Not sure if its better to hard code to (===) DRAFT only.

Comment (jasalisbury) [reply src/models/activityReport.js:283]: Do we even need this check here? Can we just let `creatorNameWithRole` always get called and if the report doesn't have a role one isn't added?

**Review by GarrettEHill — APPROVED** [review]

**Review by jasalisbury — APPROVED** [review]
:+1:

File: src/models/activityReport.js (hunk 1 above)
Comment (AdamAdHocTeam) [reply src/models/activityReport.js:282]: Will keep as is for now we will remove once we run a migration.

**Pull request merged**